endaq/ide/info.py

```diff
@@ -319,10 +319,10 @@ def to_pandas(
         t = t + np.datetime64(channel.dataset.lastUtcTime, "s")
     elif time_mode != "timedelta":
         raise ValueError(f'invalid time mode "{time_mode}"')
+    
+    if hasattr(channel, "subchannels"):
+        columns = [sch.name for sch in channel.subchannels]
+    else:
+        columns = [channel.name]
```

Comment:
It might be safer to do `if isinstance(channel, idelib.dataset.SubChannel):` (`SubChannel` is a subclass of `Channel`) instead of `hasattr()`, but it probably won't be an issue.

Reply:
that'd also work.

though I can't think of a situation where, between the existing code and the suggested code, one would fail and the other would succeed (unless we're passing in something other than the channel objects from idelib). And without such an example, I'd rather leave the code as-is just cuz I don't really want to put in another commit.